torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -125,38 +269,63 @@ void insertMkldnnPrePackedOps(script::Module& module) {
   }
 }
 
-void FuseReluWithPackedOps(std::shared_ptr<Graph>& graph) {
+template <typename T>
+void RewriteEltwiseGraph(
+    std::shared_ptr<Graph>& graph,
+    const std::map<std::string, T>& fusion_attr_map,
+    std::string prepack_op_name,
+    std::string run_op_name,
+    std::string graph_input,
+    std::string prepack_input) {
   auto conv_op_rstring = at::jit::CodeTemplate(R"(
-    graph(%input, %weight, %bias, %stride:int[], %padding:int[],
-          %dilation:int[], %groups:int, %input_size:int[], %dummy_attr:str):
-        %packed_weight_bias = mkldnn_prepacked::conv2d_prepack(
-            %weight, %bias, %stride, %padding, %dilation, %groups,
-            %input_size, %dummy_attr)
-        %conv2d_res = mkldnn_prepacked::conv2d_run(%input, %packed_weight_bias)
-        %res = aten::${op}(%conv2d_res)
+    graph(${graph_input} 
+          %input_size:int[], %dummy_attr:str, %dummy_scalars: Scalar?[], %dummy_algorithm: str?${op_input_str}):
```

Comment:
It is better to replace `dummy` with `placeholder`. For example, `dummy_attr` => `attr_placeholder`

Reply:
Renamed to `xxx_placeholder`.